tools/failures.py

```diff
@@ -182,7 +182,79 @@ def analyze_failures(start_date, end_date, testmode, ignore_failure, method):
 
 if __name__ == "__main__":
     options = parse_args()
+    # S1 - Create/update runnablejobs.json from
+    #      https://index.taskcluster.net/v1/task/gecko.v2.%s.latest.firefox.decision/
+    #
+    # Date structure:
+    #  {
+    #    "namespace": "gecko.v2.mozilla-inbound.latest.firefox.decision",
+    #    "taskId": "Dh9ZvFk5QCSprJ877cgUmw",
+    #    "rank": 0,
+    #    "data": {},
+    #    "expires": "2017-10-06T18:30:18.428Z"
+    #  }
+    #
+    # With the taskId we get to
+    # https://treeherder.mozilla.org/api/project/mozilla-inbound/runnable_jobs/?decision_task_id=Pp7ZxoH0SKyU6wnhX_Fp0g&format=json
+    #
+    # It stores the latest date from runnable apis into ~/.mozilla/seta/runnablejobs.json
+    # The contents are similar to the structure below ('meta' and 'results' keys)[1]
+    #
+    # With the data from the runnable api we then add information to the job priority table.
+    #
+    # [1]
+    # {
+    #  meta: {
+    #    count: 2317,
+    #    offset: 0,
+    #    repository: mozilla-inbound,
+    #    timetamp: 1507262400.0
+    #  },
+    #  results: [
+    #    {
+    #      build_architecture: x86_64,
+    #      build_os: win,
+    #      build_platform: windows8-64,
+    #      build_platform_id: 45,
+    #      build_system_type: buildbot,
+    #      job_coalesced_to_guid: null,
+    #      job_group_description: "",
+    #      job_group_id: 24,
+    #      job_group_name: W3C Web Platform Tests,
+    #      job_group_symbol: W,
+    #      job_type_description: "",
+    #      job_type_id: 197,
+    #      job_type_name: W3C Web Platform Tests,
+    #      job_type_symbol: 1,
+    #      machine_platform_architecture: x86_64,
+    #      machine_platform_id: 45,
+    #      machine_platform_os: win,
+    #      option_collection_hash: 32faaecac742100f7753f0c1d0aa0add01b4046b,
+    #      platform: windows8-64,
+    #      platform_option: debug,
+    #      ref_data_name: Windows 8 64-bit mozilla-inbound debug test web-platform-tests-1,
+    #      result: runnable,
+    #      state: runnable
+    #    },
+    #	 {
+    #	   "build_platform": "linux64",
+    #	   "build_system_type": "taskcluster",
+    #	   "job_coalesced_to_guid": null,
+    #	   "job_group_name": "Reftests executed by TaskCluster",
+    #	   "job_group_symbol": "tc-R",
+    #	   "job_type_description": "Reftest run",
+    #	   "job_type_name": "desktop-test-linux64/opt-reftest-8",
+    #	   "job_type_symbol": "R8",
+    #	   "platform": "linux64",
+    #	   "platform_option": "opt",
+    #	   "ref_data_name": "desktop-test-linux64/opt-reftest-8",
+    #	   "result": "runnable",
+    #	   "state": "runnable"
+    #	 },
```

Comment:
this data is not relevant to failures.py- failures.py only care about buildtype,testtype,platform from a sql query.